loadgen/cmd/otelbench/flags.go

```diff
@@ -104,6 +108,29 @@ func Init() error {
 	flag.BoolVar(&Config.Metrics, "metrics", true, "benchmark metrics")
 	flag.BoolVar(&Config.Traces, "traces", true, "benchmark traces")
 
+	// `concurrency` is similar to `agents` config in apmbench
+	// Each value passed into `concurrency` list will be used as loadgenreceiver `concurrency` config
+	Config.ConcurrencyList = []int{1} // default
+	flag.Func("concurrency", "comma-separated `list` of concurrency (number of simulated agents) to run each benchmark with",
```

Comment:
[to reviewer] `concurrency` is a good name within the scope of loadgenreceiver, but in the context of `otelbench`, it may be mistaken for the _concurrency_ of running benchmarks, e.g. running 2 benchmark cases at a time, as opposed to number of simulated agents in each benchmark. Will you think that `-agents` is a better name in the context of `otelbench`?